src/main/java/org/team340/robot/subsystems/GooseneckRollers.java

```diff
@@ -0,0 +1,70 @@
+package org.team340.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.CANdi;
+import com.ctre.phoenix6.hardware.TalonFXS;
+import edu.wpi.first.wpilibj2.command.Command;
+
+import static edu.wpi.first.wpilibj2.command.Commands.*;
+
+import java.util.function.DoubleSupplier;
+import org.team340.lib.util.command.GRRSubsystem;
+import org.team340.robot.Constants.RobotMap;
+
+public class GooseneckRollers extends GRRSubsystem {
+
+    final double kIntakeSpeed = 0.0;
+    final double kScoreSpeed = 0.0;
+    final double kIndexingSpeed = 0.0;
```

Comment:
```suggestion
    private static final TunableDouble kIntakeSpeed = Tunable.doubleValue("gooseneckRollers/kIntakeSpeed", 0.0);
    private static final TunableDouble kScoreSpeed = Tunable.doubleValue("gooseneckRollers/kScoreSpeed", 0.0);
    private static final TunableDouble kIndexingSpeed = Tunable.doubleValue("gooseneckRollers/kIndexingSpeed", 0.0);
```
Constants should be static and tunable